tests/unit/test_mixins.py

```diff
@@ -377,6 +373,85 @@ def test_is_admin_grader(self, preferences, expected_is_admin_grader):
 
 
 @ddt.ddt
+class TestAuthXBlockMixin(TestCase, TestWithPatchesMixin):
+
+    USER_ID = 1234
+    COURSE_ID = 4321
+
+    class AuthXBlockMixinGuineaPig(AuthXBlockMixin):
+
+        @property
+        def course_id(self):
+            return 4321
+
+        @property
+        def see_dashboard_for_all_orgs_perms(self):
+            return set(["all_orgs_perm"])
+
+        @property
+        def ta_roles(self):
+            return set(["ta_role"])
+
+        @property
+        def see_dashboard_ta_perms(self):
+            return set(["ta_perm"])
+
+        @property
+        def see_dashboard_role_perms(self):
+            return set(["single_role_perm"])
```

Comment:
@jbzdak extract them into a class in this module, or tests/unit/utils, or just tets/utils:

    class Constants(object):
        SINGLE_ROLE = 'single_role_perm'
        TA_PERM = 'ta_perm'

etc.